<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenDebug</class>
 <widget class="QWidget" name="ScreenDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1901</width>
     <height>1061</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="generalTab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QPushButton" name="loadScreenSubmit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="loadScreen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Screen</string>
     </property>
    </widget>
    <widget class="QComboBox" name="loadScreenSelection">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>170</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>980</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="hashTestBox">
     <property name="geometry">
      <rect>
       <x>1389</x>
       <y>10</y>
       <width>491</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Hash Test</string>
     </property>
     <widget class="QLineEdit" name="hashString">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>331</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="hashButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Hash</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hashHash">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>331</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="hashStringLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>String</string>
      </property>
     </widget>
     <widget class="QLabel" name="hashHashLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Hash</string>
      </property>
     </widget>
     <widget class="QLabel" name="hashVerifyLabel">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>80</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="arduinoDataList">
     <property name="geometry">
      <rect>
       <x>1390</x>
       <y>150</y>
       <width>491</width>
       <height>631</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>1390</x>
       <y>790</y>
       <width>491</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Arduino data</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Port name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Hasher</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextBrowser" name="portName"/>
         </item>
         <item>
          <widget class="QTextBrowser" name="hasher"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="databaseTab">
    <attribute name="title">
     <string>Database</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
